### PR TITLE
Docs: Add syntax highlighting, improved formatting & use npx

### PR DESCRIPTION
**Pull request by tobiasbueschel**

### CODE REVIEW

**Comment by mrsauravsahu** [comment]
@tobiasbueschel thanks for the contribution! have added a comment about `npx` usage.

File: readme.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ $ yarn add --dev gen-env-template
 You can now use the command like so:
 
 ```
-./node_modules/.bin/gen-env-template [path to input .env]=.env [path to output .env]=template.env
+$ npx [path to input .env]=.env [path to output .env]=.env.example
```
Comment (mrsauravsahu) [inline readme.md:56]: Let's keep this as is; and add a new section before this as `Usage via npx` - which will be 
```bash
$ npx gen-env-template [path to input .env]=.env [path to output .env]=.env.example
```

Comment (tobiasbueschel) [reply readme.md]: Updated:  [docs: add usage with npx section](https://github.com/mrsauravsahu/gen-env-template/pull/70/commits/4abf22a95373c4f1404a1c7a5bece4b436a46822)

**Comment by mrsauravsahu** [comment]
@all-contributors please add @tobiasbueschel for doc

**Pull request merged**